tests/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -328,6 +329,91 @@ class IntegrationSpec extends SpecBase with TestcontainersKafkaLike with Inside
       }
     }
 
+    "rebalance safely using transactional partitioned flow" in assertAllStagesStopped {
```

Comment:
Would it make sense to have this in `TransactionsSpec` or a new one? `IntegrationSpec` is very unspecific.

Reply:
`TransactionsSpec` works.

Reply:
The test requires a little more relaxed "patience" config due to the rebalancing.

Reply:
If needed, you can define a test case specific `PatienceConfig`.